core/src/main/java/org/keycloak/representations/idm/CredentialRepresentation.java

```diff
@@ -155,4 +155,101 @@ public void setCreatedDate(Long createdDate) {
     public void setConfig(MultivaluedHashMap<String, String> config) {
         this.config = config;
     }
+
```

Comment:
I can try to remove it by relaxing the admin event verification. Let me know what you think. Thanks